cypress/support/fragments/orders/orderDetails.js

```diff
@@ -232,4 +234,17 @@ export default {
       );
     }
   },
+
+  verifyOrderListIncludeLinkExists(linkName) {
```

Comment:
use one verb either Include or Exists, not orderList but Order or OrderRow or smthg
lise verifyOrderItemIncludesLink()\
btw this method is not used